bower.json

```diff
@@ -17,13 +17,8 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "~1.3.15",
-    "angular-animate": "^1.3.0",
-    "angular-cookies": "^1.3.0",
-    "angular-resource": "^1.3.0",
-    "angular-sanitize": "^1.3.0",
-    "angular-touch": "^1.3.0",
-    "angular-ui-router": "~0.2.13",
-    "angular-material": "~0.8.3"
+    "bootstrap-sass": "~3.3.4",
+    "react": "~0.13.1",
+    "react-bootstrap": "~0.20.0"
```

Comment:
It looks like you removed all the angular dependencies from bower yet you still reference them in HTML. This needs to be made consistent. 

Consider deleting your bower_components folder and running bower install then checking your page for any errors about missing files.